ROX-15135: Reorganize requirements structure and add recommended requirements (#61935)
openshift/openshift-docs: cloud_service/acscs-recommended-requirements.adoc

:_content-type: ASSEMBLY
[id="acscs-recommended-requirements"]
= Recommended resource requirements for {product-title-managed}
include::modules/common-attributes.adoc[]
:context: acscs-recommended-requirements

toc::[]

Recommended resource requirements provide additional details about required resources based on workload size on secured clusters.
Used resources depend on workload type and size on secured clusters. It's difficult to take one Kubernetes resource and define a sizing guide based on that. Please take into consideration that the following size recommendations are based on focused testing on a single secured cluster.

> This test creates the following objects across a given number of namespaces:

Reviewer: This part and the part after the bullets is confusing to me, because it talks about "this test" but there is no information about testing...is this referring to the testing that was done in order to come up with these guidelines? If so, may want to just say something general like "The recommended resource guidelines were developed by performing a test that created the following objects across a given number of namespaces:"

Reviewer: Although honestly this is more than the usual level of detail that I've seen given for recommended guidelines, but if you feel that it is important for the user to know this, I defer to your recommendation. Normally we would just say something like "based on our testing, we recommend x."

Author: I have applied your suggestion. I used the word "focused" test.

Inspiration for this page comes from OpenShift: https://docs.openshift.com/container-platform/4.13/scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.html#master-node-sizing_recommended-control-plane-practices

* 10 deployments, with 3 pod replicas in a sleep state, mounting 4 secrets, 4 config maps
* 10 services, each one pointing to the TCP/8080 and TCP/8443 ports of one of the previous deployments
* 1 route pointing to the first of the previous services
* 10 secrets containing 2048 random string characters
* 10 config maps containing 2048 random string characters

> During the analysis of results, the number of deployments is identified as a primary factor for increasing of used resources. And we are using the number of deployments for the estimation of required resources.

Reviewer: ```suggestion
During the analysis of results, the number of deployments was identified as a primary factor for increasing of used resources. The number of deployments was used for the estimation of required resources.
```
